app/component/itinerary/ItineraryPage.js

```diff
@@ -1096,14 +1111,24 @@ export default function ItineraryPage(props, context) {
   } else if (detailView) {
     if (naviMode) {
       content = (
-        <NaviContainer
-          itinerary={combinedEdges[selectedIndex]?.node}
-          focusToLeg={focusToLeg}
-          relayEnvironment={props.relayEnvironment}
-          combinedEdges={combinedEdges}
-          setNavigation={setNavigation}
-          mapRef={mwtRef.current}
-        />
+        <>
+          {!isNavigatorIntroDismissed && (
+            <NavigatorIntroModal
+              isOpen
+              onPrimaryClick={toggleNavigatorIntro}
+              onClose={cancelNavigatorUsage}
+              isHSL={config.CONFIG !== 'hsl'}
```

Comment:
Where did this come from? Use of word 'HSL' should be avoided in source code.

Reply:
Looks like a remnant from the time `config.navigationLogo` property was not yet being used. Will remove.